SearchService/solr_interface.py

```diff
@@ -1,48 +1,47 @@
 """ Top level functions for SOLR functions. """
 import calendar
+import urllib
+from datetime import datetime
 import logging
 import os
 import json
 import sys
 import urllib2
 
-import query_parser
-import search_exceptions
-
-from datetime import datetime
-
-from query_parser import Document
-
 from appscale.common import appscale_info
 
+import query_converter
+import search_exceptions
+from constants import (
+  HTTP_OK, INDEX_NAME_FIELD, INDEX_LOCALE_FIELD, SOLR_SERVER_PORT
```

Comment:
Not a deal breaker but we should probably replace HTTP_OK with:
httplib.OK (python 2.x)
http.HTTPStatus.OK (python 3.x)